### ISSUE TITLE
support for objects

### ISSUE THREAD

**Issue by jgallen23** [firstandthird/varson#1]
https://github.com/firstandthird/varson/blob/master/test/varson.test.js#L201

**Comment by dawnerd** [comment]
Latest version still doesn't seem to work right:

Input:
```
inputs:
  desktop:
    input:
      background: '{{color["input-background"]}}'
  desktop-wide: '{{inputs.desktop}}'
```
Outputs:
`'desktop-wide': '[object Object]'`

**Comment by jgallen23** [comment]
fixed in varson 0.3.0 and confi 2.3.0
